code/_onclick/ai.dm

```diff
@@ -48,16 +48,16 @@
 	if(!can_see(A))
 		return
 
-	if(LAZYACCESS(modifiers, SHIFT_CLICK))
-		if(LAZYACCESS(modifiers, CTRL_CLICK))
+	if(LAZYACCESS(modifiers, SHIFT_PRESS))
+		if(LAZYACCESS(modifiers, CTRL_PRESS))
 			CtrlShiftClickOn(A)
 			return
 		ShiftClickOn(A)
```

Comment:
we should rename the shiftclick procs then too

Reply:
oh yeah

Reply:
I checked those. Those procs are fine because these are called with click action.
Although the proc name isn't really nice. `ClickOn_CtrlShift` might be better than `CtrlShift_ClickOn` but I am not sure

Reply:
Hrm